libraries/chain/include/cyberway/genesis/ee_genesis_container.hpp

```diff
@@ -40,10 +40,18 @@ struct section_ee_header {
 };
 
 struct vote_ee_object {
+    account_name voter;
     int16_t weight;
     uint64_t time;
 };
 
+struct reblog_ee_object {
+    account_name account;
+    string title;
+    string body;
+    uint64_t time;
```

Comment:
fc::time_point_sec ?

Reply:
time_point_sec is 32 bit length, instead of 64 bits

Reply:
@afalaleev Cyberway uses microseconds - we need uint64_t in events. And we cannot convert it when sending events, that's because we need uint64_t in genesis.

Reply:
It is wrong.
Should be fixed in other issue

Reply:
#587